pkg/k8s/apis/cilium.io/v2/types.go

```diff
@@ -22,9 +22,9 @@ import (
 // +kubebuilder:resource:categories={cilium},singular="ciliumendpoint",path="ciliumendpoints",scope="Namespaced",shortName={cep,ciliumep}
 // +kubebuilder:printcolumn:JSONPath=".status.id",description="Cilium endpoint id",name="Endpoint ID",type=integer
```

Comment:
I might suggest dropping the EndpointID as well. This particular line is different for every node and not globally that useful.

```suggestion
```

(Please keep the `Cilium identity id`, though that should really say "Security Identity")

Reply:
Should I change `Cilium identity id` to `Security Identity`?

Reply:
Sure, I think that would be clearer given the terminology we publish on the docs.

Reply:
So it would look like this:

```
// +kubebuilder:printcolumn:JSONPath=".status.identity.id",description="Security Identity",name="Identity ID",type=integer
```

Reply:
Does that show up in the command output? I would have thought it would be the `name` field.

Reply:
I think I misunderstood. If it is shown as a column name in the output, you are right it should be inserted into the name field, it would be:
```
// +kubebuilder:printcolumn:JSONPath=".status.identity.id",description="Cilium identity id",name="Security Identity",type=integer
```

Reply:
I don't know how this works, you'll have to try it out and see.

Reply:
Ok @joestringer, I'll get on that soon

Reply:
Hi @joestringer, I have followed your feedback, and it will results:

```
NAMESPACE            NAME                                      SECURITY IDENTITY ID   ENDPOINT STATE   IPV4           IPV6
kube-system          coredns-5d78c9869d-629d7                  1867                   ready            10.244.1.226   fd00:10:244:1::4f15
kube-system          coredns-5d78c9869d-pvvb5                  1867                   ready            10.244.1.38    fd00:10:244:1::49fb
local-path-storage   local-path-provisioner-6bc4bddd6b-f8lqm   4756                   ready            10.244.1.63    fd00:10:244:1::a1ad
```

What do you think?

Reply:
Looks good to me. One nit, `SECURITY IDENTITY ID` is a bit redundant, it can just say `SECURITY IDENTITY`.

Reply:
That's great, I will change it